functoria-runtime.opam

```diff
@@ -12,11 +12,16 @@ doc:          "https://mirage.github.io/functoria/"
 license:      "ISC"
 tags:         ["org:mirage"]
 
-build: ["ocaml" "pkg/pkg.ml" "build" "--pkg-name" name "--pinned" "%{pinned}%" ]
+build: [
+  ["jbuilder" "subst" "-p" name "--name" name] {pinned}
```

Comment:
This can be simpler now, the man page says to just use `["jbuilder" "subst"] {pinned}`